### DISCUSSION TITLE
Operational command result

### DISCUSSION THREAD

**Discussion by kirillmuravyev** [PaloAltoNetworks/pan-os-python#479] [Q&A] [answered]
Hello All!
Need some help as I am new to python and pan-python\

trying to do sample command from the doc:
```
from panos import firewall
fw = firewall.Firewall('10.0.0.1', 'admin', 'mypassword')
element_response = fw.op('show system info')
print(element_response.items())
```

What I have in response is:
[('status', 'success')]

Why am I not getting all the results?
At the same time when I add "xml=True" I see whole system info

**Comment by BatD2** [comment, marked as answer]
Your response is correct. element_response.items() will return the attributes of the root element. You can navigate element tags using ElementTree methods, for example: 

`element_response.find('./result/system/hostname').text`

To view the full xml response, you can use dump() or tostring():

`ET.dump(element_response)`
